bugwarrior/services/jira.py

```diff
@@ -13,6 +13,7 @@ class JiraIssue(Issue):
     URL = 'jiraurl'
     FOREIGN_ID = 'jiraid'
     DESCRIPTION = 'jiradescription'
+    ESTIMATE = 'estimate'
```

Comment:
Unfortunately UDAs are global, so we should really do what we can to not pollute the user's UDAs with service-specific ones.  Could I convince you into changing this to `jiraestimate`?